components/forms/PaypalButton.js

```diff
@@ -83,11 +83,12 @@ class PaypalButton extends React.Component {
 export default scriptLoader('https://www.paypalobjects.com/api/checkout.js')(PaypalButton);
 
 PaypalButton.propTypes = {
-  total: PropTypes.number,
+  total: PropTypes.string,
```

Comment:
`total` is [calculated](https://github.com/texas-justice-initiative/website-nextjs/blob/master/pages/donate.js#L82) with [`Number.prototype.toFixed()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed), which returns a string. (Generally [it's not safe to use floats for money](https://husobee.github.io/money/float/2016/09/23/never-use-floats-for-currency.html), but I'd be down to revisit this in a future PR.)